### ISSUE TITLE
Advanced support for CPU usage and temperature output

### ISSUE THREAD

**Issue by Algorithm0** [raghulkrishna/deepcool-ak620-digital-linux#7]
Hello! It would be quite interesting to know how your mode works. Can you tell me on what principle you leave packets of bytes that are sent to write to the device? That would be great.
I want to expand support for those devices that output GPU information.
Perhaps it would be cool if we contacted some messenger to discuss further project expansion options.

**Comment by raghulkrishna** [comment]
hi yes that would be great for you have any suggestions

**Comment by Algorithm0** [comment]
@raghulkrishna, Can you tell me how you found what bytes to send when? I saw you insert bytes by decimal digits. It seems strange to me the distribution of the value of four numbers.

**Comment by raghulkrishna** [comment]
i used similar package like this to capture the hid data using windows os 

https://github.com/todbot/win-hid-dump

**Comment by Algorithm0** [comment]
Yes, it's good, but I don't have the devices needed so that it can be checked...

**Comment by raghulkrishna** [comment]
for which cooler you want to do the test?
